src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private final String[] keys = new String[]{"supply", "buy"};
+    private final int[] values = new int[keys.length];
+
     public void getStatistic(String fromFileName, String toFileName) {
+        reset();
+        readFromFile(fromFileName);
+        String output = calculateResult(values);
+        writeToFile(output, toFileName);
+    }
+
+    private void readFromFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
+            while (line != null) {
+                String[] keyValuePair = line.split("\\,");
```

Comment:
make separator constant field 
```suggestion
                String[] keyValuePair = line.split(",");
```